server/chat-plugins/help.ts

```diff
@@ -0,0 +1,473 @@
+/**
+ * PS Help room auto-response plugin.
```

Comment:
At a higher level, why is this a 'Help' room plugin instead of being a generic `AutoResponder` plugin that can be used in any room? Obviously it is most desirable in main's PS main's Help room, but other rooms (for example PS main's Development room) might want similar features, and coding it generically as an `AutoResponder` removes my concerns about main-specific code being checked in?

Reply:
I mean, no other room has use for it. I can make it generic, i guess, but that frankly doesn't seem useful.
Also, memory note, storing debug data has use for the help room but doing it for every room would be too much, i think.

Reply:
I don't feel this is resolved :)

@Zarel PTAL

Reply:
I feel like anyone who wants to make it generic is free to do so, but I'm not going to make Mia do it, and especially not blocking it from being useful in Help.

Reply:
I feel making it generic, due to the nature of some of the constants, would either make it risker and less useful in Help, or less useful in other rooms. I'm personally not a fan.

Reply:
I'd be open to giving it some weeks of live testing and then seeing how things go, and then coming back to making it general, but for now, i feel it'd probably inhibit it in help to make it generalized right now.